commit ce7ccd5565cf7a9226989bdea2c78de57b207af4
Author: Benda Xu <heroxbd@gentoo.org>
Date:   Mon Jun 8 18:49:57 2015 +0900

    Increase version number to avoid spurious warning.
    
    The warning happens when reading a new file (v5.34/21) in an older version of ROOT (v5.32 for example)
    (i.e. forward compatibility).  The Warning is due to the typedef now be stripped away in checksum
    calculation.

diff --git a/core/base/inc/TAttMarker.h b/core/base/inc/TAttMarker.h
index e0994e7..16d6828 100644
--- a/core/base/inc/TAttMarker.h
+++ b/core/base/inc/TAttMarker.h
@@ -53,7 +53,7 @@ public:
    virtual void     SetMarkerStyle(Style_t mstyle=1) { fMarkerStyle = mstyle;}
    virtual void     SetMarkerSize(Size_t msize=1)    { fMarkerSize  = msize;}
 
-   ClassDef(TAttMarker,2);  //Marker attributes
+   ClassDef(TAttMarker,3);  //Marker attributes
 };
 
    enum EMarkerStyle {kDot=1, kPlus, kStar, kCircle=4, kMultiply=5,
